### ISSUE TITLE
[bug] Typing ), ] or } before an existing ), ] or } respectively does not behave consistently

### ISSUE THREAD

**Issue by vskorjanc** [joeyuping/quick_latex_obsidian#27]
Hello!

I noticed that typing a closing bracket before an existing closing bracket does not always move the cursor, but sometimes adds an additional closing bracket.

## Example
```
\\ Typing {a} yields:
{a}}
\\ a new closing bracket was added, the cursor is between the two brackets.
\\ Typing {a} again yields:
{a}{a}}
\\ this time typing the closing bracket just moved the cursor and did not add an additional bracket.
\\ The cursor is between the two adjacent closing brackets. 
```
## Additional info
- Obsidian v0.13.19
- OS: Ubuntu v.20.04.3

I turned off all the community plugins except this one when testing

P.S.
Thank you for this awesome plugin, it removes a lot of friction from writing math formulae!

**Comment by joeyuping** [comment]
Thank you so much!  @vskorjanc 

I tried added some fix to this issue~ please update to 2.1.3 and take a look~

**Comment by vskorjanc** [comment]
Looks good to me, thank you very much @joeyuping !
